kubernetes/assets/dashboards/kubernetes_clusters.json

```
@@ -3790,11 +3790,11 @@
                                     "display_type": "line",
                                     "metadata": [
                                         {
-                                            "alias_name": "network writes per node",
-                                            "expression": "sum:kubernetes.io.write_bytes{$scope,$cluster} by {kube_replica_set,host}-avg:kubernetes_state.replicaset.replicas_ready{$scope,$cluster} by {host}"
+                                            "alias_name": "disk writes per node ",
```

I'm agree that `network` is the right description, but `disk` is not too. WDYT if we replace it by `IO` ?

```suggestion
                                            "alias_name": "IO writes per node ",
```

---

Ok seems fair to me.